app/presenters/concerns/assessment_tasks/assess_against_legislation_presenter.rb

```diff
@@ -3,14 +3,15 @@
 module AssessmentTasks
   extend ActiveSupport::Concern
 
-  class AssessAgainstLegislationPresenter < PlanningApplicationPresenter
+  class AssessAgainstLegislationPresenter
```

Comment:
Still need to do some work on the validation request presenters not to inherit from `PlanningApplicationPresenter`